galactus.libraries.yml

```diff
@@ -6,40 +6,53 @@ global:
     theme:
       css/clf.drupal.css: {}
   js:
-    https://cdn.ubc.ca/clf/7.0.4/js/ubc-clf.min.js: {}
     js/lib/bootstrap/alert.js: {}
   dependencies:
     - core/jquery
 
 ### CDN CLF Versions ###
+cdn-clf-js:
```

Comment:
Explanation: add CDN js as a separate library to simplify and reduce duplication